pkg/graph/resolver.go

```diff
@@ -0,0 +1,19 @@
+package graph
+
+import "github.com/cmsgov/easi-app/pkg/storage"
+
+//go:generate go run github.com/99designs/gqlgen
+
+// This file will not be regenerated automatically.
+//
+// It serves as dependency injection for your app, add any dependencies you require here.
+
+// Resolver is a resolver.
+type Resolver struct {
+	store *storage.Store
```

Comment:
I'm definitely still interested in where this would put the business logic authorization responsibilities, if we're hooking the transport layer (graphQL) straight up to the data layer.